README.md

```diff
@@ -79,8 +79,15 @@ xcode-select --install
 See the full API documentation in the [API readme](./docs/api.md) file.
 
 ## Parsing
-If you're interested in how we parse run files from different programs, check out the [parsing
-readme](./docs/parsing.md).
+Split I/O uses [LiveSplit Core][livesplit-core] for parsing. The parser is located in `lib/parser/*`.
```

Comment:
Slight nitpick: I prefer livesplit-core.

Reply:
Makes sense, wasn't sure what the exact capitalization should be since some places use LiveSplit and other livesplit.